ragthoven/tools/example_tool.py

```diff
@@ -0,0 +1,28 @@
+"""
+Every tool is a function that takes the data row in form of dictionary in a variable `args` and can modify it.
+- Every tool must return the said args in the end.
+- The tools are called sequentially as specified in yaml config.
+"""
+
+
+def fizzbuzz(args):
+    text = str(args["article"])
+    n_of_spaces = text.count(" ")
+    args["fizzbuzz"] = (
+        "fizzbuzz"
+        if n_of_spaces % 3 and n_of_spaces % 5
+        else "buzz" if n_of_spaces % 3 else "fizz" if n_of_spaces % 5 else "Nothing"
+    )
+    return args
+
+
+def count_ands(args):
+    text = str(args["article"])
+    args["and_countes"] = text.count("and")
+    return args
```

Comment:
@gkaretka does this mean one function can effectively override the results of the others?

Reply:
Yes